### DISCUSSION TITLE
Plans on a making a new PicoADK version with the RP2350?

### DISCUSSION THREAD

**Discussion by Cinezaster** [DatanoiseTV/PicoDSP-Hardware#9] [General]
Hi,

Are you planning on a making a new PicoADK version with the RP2350?

Kind regards

**Comment by DatanoiseTV** [comment]
Yes, currently working on it. First prototypes will be coming soon and I've added a branch here with the modifications.
It will use floating point math.

**Reply by Cinezaster** [reply]
Would it have the same pinout? So it can be swapped with the RP2040 verison, without redesigning my pcb.
Having a little 303 style synth project that could use some extra power and floating point math for the filter. Was planning on switching to ESP32 although the RP2350 might do the trick.
Let me know If you have something to test.

**Reply by Cinezaster** [reply]
Hi, any update on the V2 PicoADK hardware with the RP2350?

**Reply by jasonolan** [reply]
Same question. Any updates on V2 PicoADK hardware with the RP2350?
